tests/qiskit_qir/test_complex_circuits.py

```
@@ -0,0 +1,465 @@
+from qbraid_qir.qiskit.exceptions import QiskitConversionError
+
+
+def _get_body(module):
+    func = next(filter(is_entry_point, module.functions))
+    lines = str(func).splitlines()[2:-1]
+    return [line.strip() for line in lines]
+
+
+def _gate_ops(module):
+    """Return only qis__ gate call lines from the body."""
+    return [l for l in _get_body(module) if "qis__" in l]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)